client/app/pages/users/list.html

```diff
@@ -1,24 +1,47 @@
 <settings-screen>
   <div class="row">
     <div class="col-md-12">
-      <p ng-if="$ctrl.currentUser.hasPermission('admin')">
-        <a href="users/new" class="btn btn-default"><i class="fa fa-plus"></i> New User</a>
-      </p>
+      <div class="clearfix m-b-10">
+        <a ng-if="$ctrl.currentUser.hasPermission('admin')"
+          href="users/new" class="btn btn-default pull-left"
+        ><i class="fa fa-plus"></i> New User</a>
+
+        <div class="btn-group pull-right">
```

Comment:
Only admins should be able to see disabled users (at least in the UI, it's OK it comes back in the API).